### DISCUSSION TITLE
Android ssh over bluetooth

### DISCUSSION THREAD

**Discussion by jargule** [ev3dev/ev3dev#1648] [Q&A]
Has anyone tried to connect an Android phone via Bluetooth tether to a raspberry or some other Linux based system to create a PAN network? This way a SSH connection could possibly be established. Android does allow its internet connection to be shared over Bluetooth but what about the other way around?

**Comment by dlech** [comment]
It's been a long time since I tried this, but most phones should be able to provide "tethering" via Bluetooth (in other words, providing Internet connection sharing over Bluetooth). If you turn this on first, then pair the EV3 with your phone, it should be possible to access the Internet from the EV3 and use SSH between the Android device and the EV3.
